internal/stores/sql.go

```diff
@@ -113,6 +115,14 @@ func NewSQLStore(conn gorm.Dialector, migrate bool, persistInterval time.Duratio
 		lastAnnouncementSave: time.Now(),
 		persistInterval:      persistInterval,
 	}
+
+	if applyDefaultBlocklist {
+		for _, entry := range []string{"siacentral.ddnsfree.com", "siacentral.mooo.com", "51.158.108.244"} {
```

Comment:
@lukechampine any thoughts on this? Basically we considered shipping a sane list of blocklisted hosts which need to be enabled explicitly through a flag.

Reply:
it's enabled by default

Reply:
I don't think the Foundation should be deciding who gets blacklisted, even if it's disabled by default.
A good alternative would be to put a URL in the config that points to a dynamic blacklist. Then community members like SiaStats could maintain such a list. We could hard-code a list of blacklist providers (like how Embarcadero can optionally fetch market rates from SiaStats), as well as allowing the user to enter a custom URL.
For now, let's leave it out.

Reply:
Figured, that makes sense. I figured we could temporarily keep the hardcoded list until we release an alpha or something but we can just remove it for now.